tests/rados/stretch_cluster.py

```diff
@@ -628,11 +628,12 @@ def setup_crush_rule_with_no_affinity(node, rule_name: str) -> bool:
     """
     rule = rule_name
     rules = """id 11
-    type replicated
-    step take default
-    step choose firstn 0 type datacenter
-    step chooseleaf firstn 2 type host
-    step emit"""
+type replicated
+min_size 1
+max_size 10
+step choose firstn 0 type datacenter
+step chooseleaf firstn 2 type host
+step emit"""
```

Comment:
Are these line incorrectly indented?

Reply:
No, these lines are now indented as expected. All the lines part of formatted string as arranged as it would be inside the CRUSH file